### PR TITLE
LI-4 Added SwiftFormat to project

### LINKED ISSUE #4: Let's add a formatter
It would be nice to have a formatter to automatically clean up the code to reduce the time spent on code review and keeping the same style all over the project.

### PR DESCRIPTION
**Pull request by vkondrashkov**
## Status
 
ready

## Issue
 
resolves #4

### CODE REVIEW

File: Source/AnimationMaker.swift [hunk 1]
```diff
@@ -60,7 +60,7 @@ public final class AnimationMaker {
         return animations
     }
 
-    /// TODO: need to change property after animation completion
+    // TODO: need to change property after animation completion
     /// to make animation sequences relatively to each other
```
Comment (vkondrashkov) [inline Source/AnimationMaker.swift]: This seems weird, so what we will do? Use double-slash comments or disable swiftformat for this rule?

Comment (pkondrashkov) [reply Source/AnimationMaker.swift]: Cant we configure it? I think we can, need to check it though

File: Source/AnimationQueue.swift [hunk 2]
```diff
@@ -40,8 +40,9 @@ final class AnimationQueue: NSObject {
 }
 
 // MARK: - CAAnimationDelegate implementation
+
 extension AnimationQueue: CAAnimationDelegate {
-    func animationDidStop(_ anim: CAAnimation, finished flag: Bool) {
```
Comment (pkondrashkov) [inline Source/AnimationQueue.swift]: This rule should be disabled

File: Source/AnimationValue.swift [hunk 3]
```diff
@@ -8,18 +8,18 @@
 
 import UIKit
 
-public protocol AnimationValue { }
```
Comment (pkondrashkov) [inline Source/AnimationValue.swift]: Check this one. Would be cool to leave the space

File: Source/EasingMaker.swift [hunk 4]
```diff
@@ -56,7 +57,7 @@ private extension EasingMaker {
             toValue * -0.044,
             toValue * 0.1312,
             toValue * -0.3706,
-            toValue
+            toValue,
```
Comment (pkondrashkov) [inline Source/EasingMaker.swift]: We should remove this rule

File: Source/EasingType+CAMediaTimingFunction.swift [hunk 5]
```diff
@@ -9,7 +9,7 @@
 extension EasingType {
     var timingFunction: CAMediaTimingFunction? {
         switch self {
-        case .custom(let timingFunction):
+        case let .custom(timingFunction):
```
Comment (pkondrashkov) [inline Source/EasingType+CAMediaTimingFunction.swift]: Check what's wrong with this pattern matching rule please

File: Source/AnimationMaker.swift (hunk 1 above)
Comment (vkondrashkov) [reply Source/AnimationMaker.swift]: Unfortunately it happens due to `todos` rule, but we can't configure it in detail, so double-slash is preferred for `// TODO` and `// MARK`

**Pull request merged**